components/grid/toolbar.md

```diff
@@ -10,282 +10,175 @@ position: 45
 
 # Grid Toolbar
```

Comment:
The article does not mention the old ToolbarTemplate at all. I believe we still have it, so it should be documented too, even if we aim to deprecate it, which I am not aware of. Outline the differences between the two and when to use each.